src/components/RedigerAvansertBeregning/RedigerAvansertBeregning.tsx

```diff
@@ -9,12 +9,12 @@ import { EndreInntekt } from '@/components/EndreInntekt'
 import { EndreInntektVsaPensjon } from '@/components/EndreInntektVsaPensjon'
 import { InfoModalInntekt } from '@/components/InfoModalInntekt'
 import {
-  useTidligsteHelUttaksalderQuery,
-  useTidligsteGradertUttaksalderQuery,
+  useTidligstMuligHelUttakQuery,
```

Comment:
HelUttak -> HeltUttak
(for å være pirkete :)

Reply:
Takk! Bedre når det er grammatisk riktig ja :)